core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -172,7 +174,7 @@ object Defaults {
   val LeaderImbalanceCheckIntervalSeconds = 300
   val UncleanLeaderElectionEnable = false
   val InterBrokerSecurityProtocol = SecurityProtocol.PLAINTEXT.toString
-  val InterBrokerProtocolVersion = ApiVersion.latestVersion.toString
+  val InterBrokerProtocolVersion = MetadataVersion.latest.toString
```

Comment:
This should be calling `version()` not `toString()`. The `toString()` will be the enum name (IBP_3_3_IV0) which we don't want here